src/Cake.Svn/SvnAliases.Info.cs

```diff
@@ -0,0 +1,255 @@
+        ///
+        ///     Verbose("The file is a working copy: {0}", IsWorkingCopy);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Info")]
+        [CakeNamespaceImport("Cake.Svn.Info")]
+        public static bool IsWorkingCopy(this ICakeContext context, FilePath filePath)
+        {
+            var svnInformation = new SvnInfo(context.Environment, SvnClientFactoryMethod);
+
+            return svnInformation.IsWorkingCopy(filePath);
+        }
+
+        /// <summary>
+        /// Gets Subversion information about the directory at <paramref name="directoryPath"/>.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="directoryPath">The directory.</param>
+        /// <returns>The Subversion information.</returns>
+        /// <example>
+        /// <para>Gets Subversion information about the directory at <paramref name="directoryPath"/>.</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var localDirectoryInfo = GetDirectoryInfo(@"C:\project\src\");
+        ///
+        ///     foreach(var svnInfoResult in localDirectoryInfo)
+        ///     {
+        ///         Verbose("Path: {0}", svnInfoResult.Path);
+        ///     }
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Info")]
+        [CakeNamespaceImport("Cake.Svn.Info")]
+        public static IEnumerable<SvnInfoResult> GetDirectoryInfo(this ICakeContext context, DirectoryPath directoryPath)
+        {
+            return GetDirectoryInfo(context, directoryPath, null);
+        }
+
+        /// <summary>
+        /// Gets Subversion information about directory at <paramref name="directoryPath"/>.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="directoryPath">The directory.</param>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The Subversion information.</returns>
+        /// <example>
+        /// <para>Gets Subversion information about directory at <paramref name="directoryPath"/>.</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var svnInfoSettings = new SvnInfoSettings
+        ///     {
+        ///         Depth = SvnDepth.Unknown;
+        ///     };
+        ///
+        ///     var localDirectoryInfo = GetDirectoryInfo(@"C:\project\src\", svnInfoSettings);
+        ///
+        ///     foreach(var svnInfoResult in localDirectoryInfo)
+        ///     {
+        ///         Verbose("Path: {0}", svnInfoResult.Path);
+        ///     }
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Info")]
+        [CakeNamespaceImport("Cake.Svn.Info")]
+        public static IEnumerable<SvnInfoResult> GetDirectoryInfo(
```

Comment:
To avoid potential naming conflicts with other addins I suggest to name this `GetSvnDirectoryInfo`